linera-rpc/tests/snapshots/format__format.yaml.snap

```diff
@@ -733,45 +736,53 @@ NodeError:
           SEQ:
             TYPENAME: CryptoHash
     13:
+      MissingCertificatesByHeights:
+        STRUCT:
+          - chain_id:
+              TYPENAME: ChainId
+          - heights:
+              SEQ:
+                TYPENAME: BlockHeight
```

Comment:
The reason Protobuf has field IDs and always-optional fields is so that you can put new fields at the end and avoid breaking compatibility.

Reply:
Right, I know. I don't think we worry about breaking compatibility right now. It doesn't mean i shouldn't follow best practices.